src/browser/api_protocol/transport_strategy/ack.ts

```diff
@@ -5,7 +5,29 @@ export class AckMessage {
     public readonly action: string = 'ack';
     public breadcrumbs?: Array<any>;
     public correlationId: number;
+    public readonly originalAction?: string;
     public payload: AckPayload | NackPayload;
+
+    constructor(breadcrumbs?: Array<any>, originalAction?: string) {
```

Comment:
Since both parameters are optional, it would probably be best to pass them in an object
Same with `addBreadcrumb` method